packages/unit-jest/src/templates/swc/jest.config.mjs

```diff
@@ -0,0 +1,19 @@
+import { quasarEsModulesPackageNames } from '@quasar/quasar-app-extension-testing-unit-jest/jest-preset.mjs';
+
+/** @type {import('jest').Config} */
+export default {
+  preset: '@quasar/quasar-app-extension-testing-unit-jest',
+  // collectCoverage: true,
+  // coverageThreshold: {
+  //   global: {
+  //      branches: 50,
+  //      functions: 50,
+  //      lines: 50,
+  //      statements: 50
+  //   },
+  // },
+  transform: {
+    [`^(${quasarEsModulesPackageNames}).+\\.js$`]: 'babel-jest',
```

Comment:
Can we skip using babel-jest altogether and use `@swc/jest` for Quasar dependencies too? That would be a big plus, as we currently have to transpile Quasar ESM prod build when running Jest tests and Babel is pretty slow in doing that

Reply:
I'm not familiar with all the quasar build output so not sure what to point to. All I can say at this moment is that after commenting out this line, there is no performance difference (although it does still work)

Reply:
Tested on another machine... SWC might be slightly faster here (10%?) but not significant. It's definitely not slower and seems to work so I'll remove the babel-jest on next commit